projects/eslint-config/aui.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+'use strict';
+
+const local = require('./utils/local');
+
+const config = {
+	extends: [require.resolve('./react')],
+	plugins: [local('@liferay/portal')],
```

Comment:
Are these two configs necessary?

Reply:
Probably not, I don't remember adding these 🤔